packages/circom/default.nix

```diff
@@ -4,15 +4,17 @@
 }:
 rustPlatform.buildRustPackage rec {
   pname = "circom";
-  version = "2.0.6";
+  version = "2.0.4";
 
   src = fetchFromGitHub {
     owner = "iden3";
     repo = "circom";
-    rev = "v${version}";
-    sha256 = "sha256-T6kRa4TCIoJh7zHLanOb2w6WcIVapF6apblxjg5OxIk=";
+    rev = "47883d98c45fe25c6d010f16b717cca6d0dea745";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
Unfortunately, we need to work with specific circom commit.
Because even with a patch applied the build for `circom-pairing` still fails
Explanation: https://github.com/yi-sun/circom-pairing/issues/11#issuecomment-1211069201